DotNet.DocsTools/GitHubObjects/QuestIssue.cs

```diff
@@ -1,227 +1,30 @@
-﻿using DotNetDocs.Tools.GitHubCommunications;
-using Microsoft.DotnetOrg.Ospo;
-using System.Text.Json;
+﻿using System.Text.Json;
+using DotNetDocs.Tools.GitHubCommunications;
 
 namespace DotNet.DocsTools.GitHubObjects;
-
```

Comment:
Here, almost all the removed code was refactored into the new base class `QuestIssueOrPullRequest`.